databricks/koalas/missing/indexes.py

```diff
@@ -104,7 +104,6 @@ class _MissingPandasLikeIndex(object):
     to_numpy = unsupported_function('to_numpy')
     transpose = unsupported_function('transpose')
     union = unsupported_function('union')
-    unique = unsupported_function('unique')
     value_counts = unsupported_function('value_counts')
     view = unsupported_function('view')
     where = unsupported_function('where')
```

Comment:
@charlesdong1991, sorry can you remove `unique` at `_MissingPandasLikeMultiIndex` as well? 

Techinically `MultiIndex` has the method called `unique`. It's a bit complicated because `MultiIndex` inherits `Index`, so I suggested to throw an exception :-)..

Reply:
ok, removed! @HyukjinKwon 

i will get some free time recently, and see if I could add some properties and functions for `Index` and `MultiIndex` and then implement `unique` for `MultiIndex` if itholic hasn't implemented them all beforehand xD 😝

Reply:
That would be awesome!!